recipes/ncl/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set version = "6.3.0" %}
+
+package:
+    name: ncl
+    version: {{ version }}
+
+source:
+    fn: ncl-{{ version }}.tar.gz
+    url: https://github.com/NCAR/ncl/archive/{{ version }}.tar.gz
+    sha256: d2cd8a1de1c498c1832f0765113ad58086d1d3b29bf5d2cc9b1ba60f1919951c
+
+build:
+    number: 0
+    skip: True  # [win]
+    detect_binary_files_with_prefix: true
+
+requirements:
+    build:
+        - gcc
+        - libnetcdf 4.4.*
+        - proj4 4.9.3
+        - gdal 2.1.*
+        - g2clib 1.6.*
+        - hdfeos2
+        - hdfeos5
+        - jasper
+        - libpng
+        - jpeg 9*
+        - zlib 1.2.*
+        - hdf4
+        - hdf5 1.8.17|1.8.17.*
+        - libiconv
+        - curl
+        - cairo  # [linux]
+        - freetype  # [linux]
+        - udunits2
+        - bzip2 1.0.*
+        - gsl
+        - pkg-config
+        - pixman
+    run:
+        - libgcc
```

Comment:
To avoid issues on `OS X` we try to use

```yaml
        - libgcc  # [linux]
        - libgfortran  # [osx]
```

When Fortran (and `libquadmath`) is needed. If you also need `libgomp` then you can remove leave as is.